sdcm/tester.py

```diff
@@ -796,6 +796,11 @@ def is_encrypt_keys_needed(self):
         )
 
     def prepare_kms_host(self) -> None:
+        if self.params.is_enterprise and self.params.get('cluster_backend') == 'aws' and not self.params.get('scylla_encryption_options'):
```

Comment:
Why using that implicit way?
Why not set it in the `default` configuration?

Reply:
you'll have to explain it further, since I'm not sure what do you mean by that.

Reply:
New config option may be added to define the need to configure encryption.
In this case it will become `explicit` way.
For now, everyone will be confused to see encryption in place not having it in configuration.

Reply:
@roy WDYT? ^

Reply:
again the whole idea is to enable it by default for all of enterprise runs on AWS, I don't see how putting something in to configuration would change the confusion here.

adding one more configuration value, just for this default won't be a big change, and still gonna be confusing.